corehq/apps/reports/models.py

```diff
@@ -644,6 +644,12 @@ class FormExportSchema(HQExportSchema):
     def update_schema(self):
         super(FormExportSchema, self).update_schema()
         self.update_question_schema()
+        # update SplitColumn options
+        for column in [column for table in self.tables for column in table.columns]:
+            if isinstance(column, SplitColumn):
+                question = self.question_schema.question_schema.get(column.index)
+                column.options = question.options
+                column.ignore_extras = True
```

Comment:
I'm questioning my earlier g2g comment.

I assume this column is local to `self` (i.e., not a global)? If not, seems like mutating it here could be problematic for the same reason modifying global state is ever questionable.